ocean-client/src/utils/store.ts

```diff
@@ -53,11 +53,9 @@ export class Store {
         this.settings.maxCollateralRatio = this.getNumberValue(MaxCollateralRatioKey, parameters) ?? this.settings.maxCollateralRatio
         this.settings.LMToken = this.getValue(LMTokenKey, parameters)
 
-        if ((decryptedSeed.Parameter?.Value?.indexOf(",") ?? -1) > 0) {
-            this.settings.seed = decryptedSeed.Parameter?.Value?.split(',') ?? []
-        } else {
-            this.settings.seed = decryptedSeed.Parameter?.Value?.split(' ') ?? []
-        }
+        let seedList= decryptedSeed?.Parameter?.Value?.replace(/[ ,]+/g," ")
```

Comment:
I am bad with thinking through regex, without actually testing them. But does this correctly replace?
- abc, def, ghi
- abc ,def ,ghi

Reply:
and also double space? because below trim just removes whitespaces in front and the end of the list.

I mean we don't need to make it super fancy and work in every possible way. Just asking

Reply:
yes it does. [ ,]+ means "a character thats a space or ',' , take such a character at least once, as often as you like" and we replace that with a single " ".
so also abc , ,   , ,,, def ghi,jkl,,  would work. cause first replace all spaces and , with space and ensure no double spaces, then trim.